sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -296,6 +297,45 @@ class SessionCatalog(
   }
 
   /**
+   * Alter the schema of a table identified by the provided table identifier to add new columns
+   * @param identifier TableIdentifier
+   * @param columns new columns
+   * @param caseSensitive enforce case sensitivity for column names
+   */
+  def alterTableAddColumns(
```

Comment:
I will change back to use `alterTableSchema` to make it more generic for other schema evolution features.